googlemock/CMakeLists.txt

```diff
@@ -103,20 +103,23 @@ cxx_library(gmock_main
 # to the targets for when we are part of a parent build (ie being pulled
 # in via add_subdirectory() rather than being a standalone build).
 if (DEFINED CMAKE_VERSION AND NOT "${CMAKE_VERSION}" VERSION_LESS "2.8.11")
-  target_include_directories(gmock      SYSTEM INTERFACE "${gmock_SOURCE_DIR}/include")
-  target_include_directories(gmock_main SYSTEM INTERFACE "${gmock_SOURCE_DIR}/include")
+  set(gmock_include_dirs $<INSTALL_INTERFACE:include>)
```

Comment:
If local directory "${gmock_SOURCE_DIR}/include" is not needed then all this block can be removed and `INCLUDES DESTINATION include` used instead in `install`, see:
* https://github.com/forexample/package-example/blob/8b4b91bc6f3bc61de958e61536f63fd35a2648a3/Foo/CMakeLists.txt#L111-L118

Will achieve the same but in cleaner way. `INCLUDES DESTINATION` available since CMake 2.8.12:
* https://cmake.org/cmake/help/v2.8.12/cmake.html#command:install

Reply:
Agreed. Updated accordingly.